deploy/helm-chart/README.md

```diff
@@ -0,0 +1,135 @@
+# NGINX Kubernetes Gateway Helm Chart
+
+## Introduction
+
+This chart deploys the NGINX Kubernetes Gateway in your Kubernetes cluster.
+
+## Prerequisites
+
+- [Helm 3.0+](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+> Note: NGINX Kubernetes Gateway can only run in the `nginx-gateway` namespace. This limitation will be addressed in
+the future releases.
+
+## Installing the Chart
+
+> Note: The Gateway API resources from the standard channel (the CRDs and the validating webhook) must be installed
+before deploying NGINX Kubernetes Gateway. By default, they will be installed by the Chart if not already
+present in the cluster. If they are already installed in your cluster, please ensure they are the correct version as
+supported by the NGINX Kubernetes Gateway - [see the Technical Specifications](../../README.md#technical-specifications).
+Helm will not upgrade CRDs - to do so manually, see
+[Upgrading the Gateway API resources](#upgrading-the-gateway-resources).
+
+### Installing the Chart from the OCI registry
+
+To install the chart with the release name `my-release` (`my-release` is the name that you choose) into the
+nginx-gateway namespace (with optional `--create-namespace` flag - you can omit if the namespace already exists), and
+the Gateway API resources from the standard channel (if not already present):
+
+```
+helm install my-release oci://ghcr.io/nginxinc/charts/nginx-gateway --version 0.0.0-edge --create-namespace --wait-for-jobs -n nginx-gateway
+```
+
+### Installing the Chart via Sources
+
+#### Pulling the Chart
+
+```
+helm pull oci://ghcr.io/nginxinc/charts/nginx-gateway --untar --version 0.0.0-edge
+cd nginx-gateway
+```
+
+#### Installing the Chart
+
+To install the chart with the release name `my-release` (`my-release` is the name that you choose) into the
+nginx-gateway namespace (with optional `--create-namespace` flag - you can omit if the namespace already exists), and
+the Gateway API resources from the standard channel (if not already present):
+
+```
+helm install my-release . --create-namespace --wait-for-jobs -n nginx-gateway
+```
+
+## Upgrading the Chart
+### Upgrading the Gateway resources
+Helm does not upgrade CRDs during a release upgrade, or on an install if they are already present in the cluster.
+Before you upgrade a release, ensure the Gateway API resources are up to date by doing one of the following:
+
+1. To upgrade the Gateway resources from the NGINX Kubernetes Gateway Chart sources, pull the chart sources as described
+   in [Pulling the Chart](#pulling-the-chart) and then run:
+   ```
+   kubectl apply -f crds/
+   ```
+1. To upgrade the Gateway resources from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
```

Comment:
I feel like this option might be better as the first one, since it's less steps.